Cargo.toml

```diff
@@ -5,13 +5,15 @@ edition = "2021"
 
 [dependencies]
 anyhow = "1.0.71"
-wasmparser = "0.112.0"
-wasm-encoder = "0.32.0"
-wit-component = "0.14.0"
-wit-parser = "0.11.0"
+wasmparser = "0.113.3"
+wasm-encoder = "0.33.0"
+wit-component = "0.14.5"
+wit-parser = "0.11.3"
 
 [dev-dependencies]
-wasmtime = { version = "13.0.0", features = ["component-model"] }
-wasmtime-wasi = "13.0.0"
+wasmtime = { git = "https://github.com/bytecodealliance/wasmtime", rev = "4c34504efb258a0c51c6a5f3f8a5b24d987993b9", features = [
```

Comment:
This appears to be pointing to the `main` branch of `wasmtime`, as opposed to the `release-14.0.0` branch.  Is that intentional?  I was thinking we'd aim to use release 14 for Spin 2.0.

Reply:
cf. https://github.com/fermyon/spin-componentize/tree/wasmtime-17505667, where I used a snapshot of the `release-14.0.0` branch.

Reply:
Good call - I originally didn't do this because `release-14.0.0` is a bit older than `main`, but for `spin-componentize` all the changes I need are on `release-14.0.0`.